aws_lambda_powertools/utilities/idempotency/base.py

```diff
@@ -113,6 +116,14 @@ def _process_idempotency(self):
 
         return self._get_function_response()
 
+    def _get_remaining_time_in_seconds(self) -> Optional[int]:
+        """
+        Try to get the time remaining in seconds from the lambda context
+        """
+        if self.fn_args and len(self.fn_args) == 2 and getattr(self.fn_args[1], "get_remaining_time_in_millis", None):
```

Comment:
would it be better to change the last part to `callable(self.fn_args[1].get_remaining_time_in_millis)` ?

Reply:
We have used `getattr` is other places as `get_remaining_time_in_millis` function may not be present in the case of `@idempotent_function`.

Because then `callable(self.fn_args[1].get_remaining_time_in_millis)` will raise an error